ember-flight-icons/tests/integration/components/flight-icon-test.js

```diff
@@ -24,13 +24,17 @@ module('Integration | Component | flight-icon', function (hooks) {
     await render(hbs`<FlightIcon @name="activity" />`);
     assert
       .dom('svg.flight-icon.flight-icon-activity')
-      .hasStyle({ height: '16px', width: '16px' });
+      .hasStyle({ height: '16px', width: '16px' })
```

Comment:
Filed an issue for this diff re: tooling tweak, not impt though https://github.com/hashicorp/flight/issues/337